schedule.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+echo "$USER bb8 backup # BB8 backup" > /etc/cron.d/bb8
```

Comment:
My only comment is that if this is meant to be run via `sudo`, which will set `$USER` to `root`, should this not be `root bb8 backup`?  I had to check what the behaviour of sudo was with the USER variable.  Unfortunately it's a bit complicated:

```
     LOGNAME          Set to the login name of the target user when the -i
                      option is specified, when the set_logname option is
                      enabled in sudoers or when the env_reset option is
                      enabled in sudoers (unless LOGNAME is present in the
                      env_keep list).
...
     USER             Set to the same value as LOGNAME, described above.
```

Reply:
Good catch. Much clearer just to always run as root, since everything is being run by docker anyway.